testsuite/forge-cli/src/main.rs

```diff
@@ -589,6 +589,7 @@ fn single_test_suite(
         "gather_metrics" => gather_metrics(),
         "net_bench" => net_bench(),
         "net_bench_two_region_env" => net_bench_two_region_env(),
+        "net_bench_two_region_env_small_messages" => net_bench_two_region_env_small_messages(),
```

Comment:
Not for this PR, but at some point we'll really need to split this file up 😆

Reply:
Yeah, it's a monster...